### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by humoflife**
Updated the integration build and deployment instructions.

### What does this PR do?

A brief description of the change being made with this pull request.

Prevent user pitfalls. Agents do not retain their integrations during restart unless the agent container was built with it.

### Motivation

What inspired you to submit this pull request?

Increase user success and smoother agent build and deployment experience, and their ability to configure the agent.

### Review checklist

- [ ] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [ ] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

Anything else we should know when reviewing?

Thank you for your review. Much appreciated.

### CODE REVIEW

**Comment by estherk15** [comment]
Thanks for contributing to our docs @humoflife, a member of our documentation team will review!

File: upbound_uxp/README.md [hunk 1]
```diff
@@ -84,49 +84,296 @@ roleRef:
...
+1. Edit the `upbound_uxp.d/conf.yaml` file, in the `conf.d/` folder at the root of your Agent's configuration directory to start collecting your Upbound UXP performance data. See the [sample upbound_uxp.d/conf.yaml](https://github.com/DataDog/integrations-extras/blob/master/upbound_uxp/datadog_checks/upbound_uxp/data/conf.yaml.example) for all available configuration options.
+
+2. [Restart the Agent](https://docs.datadoghq.com/agent/guide/agent-commands/?tab=agentv6v7#start-stop-and-restart-the-agent).
+
+### Validation
+
+[Run the Agent's status subcommand](https://docs.datadoghq.com/agent/guide/agent-commands/?tab=agentv6v7#agent-status-and-information) and look for `upbound_uxp` under the Checks section.
+
+
+### Installation for Developers and Contributors
```
Comment (anawishnoff) [inline upbound_uxp/README.md:114]: Hi @humoflife, I don't beleive this section is needed. On integration tiles, we usually only specify installation instructions for customers.

**Review by nmuesch — APPROVED** [review]

File: upbound_uxp/README.md (hunk 1 above)
Comment (humoflife) [reply upbound_uxp/README.md]: This was removed.

**Review by anawishnoff — APPROVED** [review]

File: upbound_uxp/README.md [hunk 2]
```diff
@@ -84,71 +84,171 @@ roleRef:
...
-### Installation
+The Upbound UXP integration is not included in the [Datadog Agent][2] package, so you need to install it manually.
 
-To install the Upbound UXP check on your host:
+### Installation
 
+For Agent v7.21+ / v6.21+, follow the instructions below to install the check on your host. 
 
-1. Install the [developer toolkit][11] on any machine.
+1. Using the Dockerfile example in [Use Community Integrations][3], build a custom image of the Agent that includes the Upbound integration
```
Comment (drichards-87) [inline upbound_uxp/README.md:93]:
```suggestion
1. Using the Dockerfile example in [Use Community Integrations][3], build a custom image of the Agent that includes the Upbound integration.
```

**Review by drichards-87 — APPROVED** [review]
Left one tiny suggestion from Docs and approved the PR.

**Pull request merged**